### PR TITLE
Set image width and height to auto

### PR DESCRIPTION
**Pull request by epinter**
The image will be resized as the space available changes.

### CODE REVIEW

**Comment by Mikesch-mp** [comment]
HI Emerson,

what behavior you want to get from this? Resize the graph, that is in a fixed size, to make it virtually larger ? I tested your code and the only thing that is changing for me is the border around it when i do mouse over the graph.

Regards,
Carsten

File: public/css/module.less [hunk 1]
```diff
@@ -9,6 +9,10 @@
 .grafana-img {
     display: block;
     /* box-shadow: 0 3px 15px 0 rgba(0, 0, 0, 0.55) !important; */
+    width: auto;
+    height: auto;
+    max-width: 100%;
+    max-height:100%;
```
Comment (ankit167) [inline public/css/module.less]: nit, blank space after ':'

**Comment by epinter** [comment]
The image size is reduced when you reduce window size.

**Comment by Mikesch-mp** [comment]
Ok i didnt test that. Cool feature :) Will be in 1.1.6

**Pull request merged**